src/sentry/sentry_metrics/indexer/indexer_consumer.py

```diff
@@ -0,0 +1,82 @@
+import logging
+from typing import Any, Dict, Optional, Sequence
+
+from confluent_kafka import Producer
+from django.conf import settings
+
+from sentry.sentry_metrics import indexer
+from sentry.utils import json, kafka_config
+from sentry.utils.batching_kafka_consumer import AbstractBatchWorker, BatchingKafkaConsumer
+from sentry.utils.kafka import create_batching_kafka_consumer
+
+logger = logging.getLogger(__name__)
+
+snuba_metrics = settings.KAFKA_TOPICS[settings.KAFKA_SNUBA_METRICS]
+
+
+def get_metrics_consumer(
+    topic: Optional[str] = None, **options: Dict[str, str]
+) -> BatchingKafkaConsumer:
+    return create_batching_kafka_consumer({topic}, worker=MetricsIndexerWorker(), **options)
+
+
+class MetricsIndexerWorker(AbstractBatchWorker):  # type: ignore
+    def process_message(self, message: Any) -> Any:
+        parsed_message: Dict[str, Any] = json.loads(message.value(), use_rapid_json=True)
+
+        org_id = parsed_message["org_id"]
+        metric_name = parsed_message["name"]
+        tags = parsed_message["tags"]
+
+        strings = {metric_name}
+        strings.update(tags.keys())
+        strings.update(tags.values())
+
+        mapping = indexer.bulk_record(org_id, list(strings))  # type: ignore
```

Comment:
Why do we need to ignore the type checking ?

Reply:
when I run `make backend-typing` I get:
`src/sentry/sentry_metrics/indexer/indexer_consumer.py:48: error: Module has no attribute "bulk_record`

I couldn't figure out how to fix it 😞

Reply:
mh ok. I think it has to do with the way we expose methods with the service delegator.